res/autoload.template

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Yes, this is exactly the idea (probably /usr/share/php-fedora-autoloader/res/autoload.template)

Of course some package smay prefer to provide their own template.